firebase-firestore/src/main/java/com/google/firebase/firestore/AggregateField.java

```diff
@@ -86,43 +88,109 @@ public int hashCode() {
+   *   <li>A sum that overflows the maximum representable double value will result in a double return value representing infinity.
+   * </ul>
+   *
+   * @param field Specifies the field to sum across the result set.
+   * @return The `SumAggregateField` object that can be used to compute the sum of a specified field over a range of documents in the result set of a query.
+   */
   @NonNull
   public static SumAggregateField sum(@NonNull String field) {
     return new SumAggregateField(FieldPath.fromDotSeparatedPath(field));
   }
 
+  /**
+   * Create a {@link SumAggregateField} object that can be used to compute the sum of a specified field over a range of documents in the result set of a query.
+   *
+   * <p>The result of a sum operation will always be a 64-bit integer value, a double, or NaN.
+   *
+   * <ul>
+   *   <li>Summing over zero documents or fields will result in 0L.
+   *   <li>Summing over NaN will result in a double value representing NaN.
+   *   <li>A sum that overflows the maximum representable 64-bit integer value will result in a double return value. This may result in lost precision of the result.
+   *   <li>A sum that overflows the maximum representable double value will result in a double return value representing infinity.
+   * </ul>
+   *
+   * @param fieldPath Specifies the field to sum across the result set.
+   * @return The `SumAggregateField` object that can be used to compute the sum of a specified field over a range of documents in the result set of a query.
+   */
   @NonNull
   public static SumAggregateField sum(@NonNull FieldPath fieldPath) {
     return new SumAggregateField(fieldPath);
   }
 
+  /**
+   * Create an {@link AverageAggregateField} object that can be used to compute the average of a specified field over a range of documents in the result set of a query.
+   *
+   * <p>The result of an average operation will always be a 64-bit integer value, a double, or NaN.
+   *
+   * <ul>
+   *   <li>Averaging over zero documents or fields will result in a double value representing NaN.
+   *   <li>Averaging over NaN will result in a double value representing NaN.
+   * </ul>
+   *
+   * @param field Specifies the field to average across the result set.
+   * @return The `AverageAggregateField` object that can be used to compute the average of a specified field over a range of documents in the result set of a query.
+   */
   @NonNull
   public static AverageAggregateField average(@NonNull String field) {
     return new AverageAggregateField(FieldPath.fromDotSeparatedPath(field));
   }
 
+  /**
+   * Create an {@link AverageAggregateField} object that can be used to compute the average of a specified field over a range of documents in the result set of a query.
+   *
+   * <p>The result of an average operation will always be a 64-bit integer value, a double, or NaN.
```

Comment:
This should be updated to:
```
   * <p>The result of an average operation will always be a double or NaN.
```

I need to fix this in iOS as well.

Reply:
>.< I also need to fix it for Java

Reply:
I fixed it for Android and ran the api doc generation and formatting tasks that were causing @cherylEnkidu trouble.